include/deal.II/matrix_free/dof_info.templates.h

```diff
@@ -1568,9 +1568,9 @@ namespace internal
         ++n_rows;
 
       // first determine row lengths
-      std::vector<unsigned int>   row_lengths(n_rows);
-      std::vector<Threads::Mutex> mutexes(
-        n_rows / internal::bucket_size_threading + 1);
+      std::vector<unsigned int> row_lengths(n_rows);
+      std::vector<std::mutex> mutexes(n_rows / internal::bucket_size_threading +
+                                      1);
```

Comment:
I'm surprised this works, given that the mutex class is not copyable...

Reply:
This call only requires a default constructor, see https://godbolt.org/z/wQevHQ.

Reply:
I thought that `std::vector` had as one of its requirements that its template argument must be copyable. It may be that this particular call doesn't require it, though.